### ISSUE TITLE
Invalid directory - missing lib directory

### ISSUE THREAD

**Issue by rajesh2531** [streamsets/datacollector-docker#15]
Hi
I overriden USER_LIBRARIES_DIR as '/opt/streamsets-datacollector/' as given in Dockerfile. I am getting the "Invalid directory /opt/streamsets-datacollector/user-libs , missing lib directory" exception streamsets bootstrap..  

I removed the workaround code to copy the user-libs from SDC_DIST to User defined folder. Streamsets was able to bootstrap successfully. Looks like we dont need to perform this step as 'streamsets-dev-lib' is present under streamsets-libs folder for version >3.0 I believe.

**Comment by dimaspivak** [comment]
Hi Rajesh, wanna submit a pull request?

**Comment by rajesh2531** [comment]
sure will do.

**Comment by kunickiaj** [comment]
You need to re-build the image if you override that dir since its created in configure-sdc.sh and not the entrypoint script.

One fix would be to make it a build arg and make it more obvious that it requires a rebuild. Curious as to why you are interested in overriding the directory we specify?

**Comment by rajesh2531** [comment]
sorry for the confusion in my statement earlier. I did not override it. I generally mean to say that you overriden USER_LIBRARIES_DIR and not using dir which is in streamsets home folder.  And  you added workaround code to copy the dev lib folder from /opt/streamsets-datacollector/user-libs to overriden directory to avoid SDC-8005.  now its time to remove that workaround code as it causing issues while using sdc v3.2.0.0 . I think that is code is safe to remove now as I see the dev lib folder is now added to streamsets-lib fodler under sdc install directory.

**Comment by dimaspivak** [comment]
I don't think it causes issues in 3.2.0.0, though. The image we pushed to Docker Hub is what we continue to use for testing. I'm not following what issue needs solving here.
